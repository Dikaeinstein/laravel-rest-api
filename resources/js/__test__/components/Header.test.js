import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/Header';

describe('<Header />', () => {
  it('should render correctly', () => {
    const tree = shallow(<Header />);
    expect(tree).toMatchSnapshot();
  });
});
