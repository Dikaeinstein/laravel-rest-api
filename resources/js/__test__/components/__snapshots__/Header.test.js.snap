// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render correctly 1`] = `
<nav
  className="navbar navbar-expand-md navbar-light navbar-laravel"
>
  <div
    className="container"
  >
    <Link
      className="navbar-brand"
      replace={false}
      to="/"
    >
      ArticulaR
    </Link>
  </div>
</nav>
`;
